src/components/common/Field.tsx

```diff
@@ -131,3 +132,48 @@ export const SelectField: React.FC<SelectFieldProps> = ({
     </div>
   );
 };
+
+type CheckboxFieldProps = {
+  fieldClassName?: string;
+  tip?: string;
+  error?: string;
+} & CheckboxProps &
+  HTMLProps<HTMLSelectElement>;
+
+export const CheckboxField: React.FC<CheckboxFieldProps> = ({
+  fieldClassName,
+  label,
+  tip,
+  theme, // TODO: 5.0 theme, incompatible with 6.0 remove at 6.0
```

Comment:
Is this actually used?

Reply:
`label` is, but I can probably get rid of the others

Reply:
removed unneeded props